src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  align-content: space-evenly;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
```

Comment:
Be consistent with your vertical margins. It's recommended to add only top or only bottom margin, but not both to avoid margin collapse issues. Since you've set the right margin here, ensure that any top or bottom margins are handled consistently throughout your CSS.